### PR TITLE
Adds the ability to click on the notification and supply a callback to be called.

### PR DESCRIPTION
**Pull request by sethwebster**
Need the ability to click the notification (click to retry, in my case) so thought I'd submit this patch.

Let me know if you want me to squash these commits.

### CODE REVIEW

File: addon/components/notification-message.js [hunk 1]
```diff
@@ -58,6 +64,11 @@ export default Ember.Component.extend({
     }
   },
 
+  clickableClass: Ember.computed('notification.onClick', function() {
```
Comment (ynnoj) [inline addon/components/notification-message.js]: This property becomes redundant if we include the class in the `classNameBindings` array:

`'notification.onClick:c-notification--clickable'`

**Comment by ynnoj** [comment]
@sethwebster One comment, then happy to merge :rocket:

File: addon/components/notification-message.js (hunk 1 above)
Comment (sethwebster) [inline addon/components/notification-message.js]: Ah yes, of course. Will do.

**Comment by sethwebster** [comment]
Travis seems to have failed but the build passes...
image

**Comment by ynnoj** [comment]
@sethwebster Looks like the hanging tests are related to ember-cli (https://github.com/ember-cli/ember-cli/issues/5317). Please pin jQuery to `1.11.3` in `bower.json`, that should hopefully fix it :shipit:

**Comment by sethwebster** [comment]
Tried that; No dice. Not sure what's going on but I am seeing the same issue on codeship.io (on a completely different and unrelated project) with the npm hang. Same message.

**Comment by ynnoj** [comment]
@sethwebster I have fixed this issue on `master` now. If you go ahead and rebase your PR on that branch the tests should pass.

**Comment by sethwebster** [comment]
Looks like it passed now. I also squashed.

**Comment by ynnoj** [comment]
:tada:

**Comment by sethwebster** [comment]
:+1: Thanks for the addon!

**Pull request merged**